### DISCUSSION TITLE
Bounds property on DeckGL has big impact on performance

### DISCUSSION THREAD

**Discussion by geohuz** [visgl/deck.gl#6008] [Q&A] [answered]
I'm using DeckGL react component and found if using **bounds** property on heatmap component will make big impact on performance, what is the posible reason for that? Rendering becomes extremery slow if set bounds.

**Comment by Pessimistress** [comment, marked as answer]
There is no property called `bounds` in DeckGL or HeatmapLayer.

There are many things that might affect performance, https://deck.gl/docs/developer-guide/performance  is a good place to start. 

You will get better help if you include code samples with a question.
